tooling/github-configuration.md

```diff
@@ -0,0 +1,103 @@
+# GitHub InnerSource Configuration
+
+- [Enterprise Setting](#enterprise-setting)
+  - [Repository policies](#repository-policies)
+    - [Base permissions](#base-permissions)
+    - [Repository creation](#repository-creation)
+    - [Repository forking (Private / Internal)](#repository-forking-private--internal)
+- [Organization Setting](#organization-setting)
+  - [Member privileges](#member-privileges)
+    - [Base permissions](#base-permissions-1)
+    - [Repository creation](#repository-creation-1)
+    - [Repository forking](#repository-forking)
+- [GitHub Enterprise Server Setting](#github-enterprise-server-setting)
+  - [GitHub Connect - Server statistics](#github-connect---server-statistics)
+- [Resources](#resources)
+
+## Enterprise Setting
+
+### Repository policies
+
+#### Base permissions
+
+When setting up the base repository permissions to promote InnerSource within your organization on GitHub, it's important to strike a balance between transparency, collaboration, and respecting constraints and limitations. Here are some best practices for setting up repository permissions:
+
+| Permission | Description |
+| --- | --- |
+| No Policy | Inherit permissions from the organization or parent repository. Provides flexibility for projects with unique permission needs. |
+| No Permission | Not recommended if any of the repositories in that org might benefit from collaboration. Completely restricts access to the repository, hindering collaboration and knowledge sharing. Repository access requires finding and talking to org owners. |
+| Read | All organization members can view the repository's contents, promoting transparency and allowing employees to learn from each other's work. |
+| Write | Specific individuals or teams with the "Write" permission can make contributions to the repository. Grant to active InnerSource participants with relevant skills. |
+| Admin | Limited to a select group responsible for managing the repository and overseeing InnerSource initiatives. Handles repository configuration and review processes. |
+
+It is better to choose No Policy and let the user choose, but it is also a good idea to consciously choose Read permission.
```

Comment:
From @JustinGosses and @yuhattor  ([The original review comment](https://docs.google.com/document/d/1lPR90dIiWKlsNlV46xhGkR9pP0GNYmeAB7xOk-_MAYo/edit?disco=AAAA1O-4cBI))

**Yuki**
I would like to discuss regarding the recommendations here. In general, I think No Policy is better at the Enterprise level. I think there are many backgrounds. It may not be a good idea to write a "recommendation" here in the first place, since it varies from company to company.

**Justin**
Maybe just call out that establishing policies at enterprise and organization level creates significant process burden on developers for collaboration and access. Alternative approach is to establish written rules for how to make good choices but let individual decisions be made at repo level.